<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>janela</class>
 <widget class="QMainWindow" name="janela">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1097</width>
    <height>587</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1600</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 85, 0);
font: 8pt &quot;Tahoma&quot;;
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QTableWidget{
 margin:auto;
}
QPushButton:hover#parto
{
  font-size: 20px;
}
QPushButton#parto
{
  background-color: rgb(170, 170, 0);
  font-size: 16px;
}
QPushButton#todas_2
{
 background-color: rgb(255, 255, 0);
  border-radius: 0px;
  background-color: rgb(255, 255, 255);
  font-size: 16px;
}
QPushButton:hover#todas_2
{
  font-size: 20px;
}
QPushButton#calcular
{
 background-color: rgb(85, 255, 127);
 font-size: 16px;
}
QPushButton:hover#calcular
{
  font-size: 20px;
}
QPushButton#deletar
{
  background-color: rgb(255, 0, 0);
  font-size: 16px;
}
QPushButton:hover#deletar
{
  font-size: 20px;
}
QPushButton#nova_vaca
{
  background-color: rgb(0, 0, 255);
  font-size: 16px;
}
QPushButton:hover#nova_vaca
{
  font-size: 20px;
}
QPushButton#nova_vaca_2
{
background-color: rgb(255, 255, 0);
border-radius: 0px;
background: rgb(255, 255, 0);
font-size: 16px;
}
QPushButton:hover#nova_vaca_2
{
  font-size: 20px;
}
QPushButton#pesquisar
{
  background-color: rgb(255, 85, 255);
  font-size: 16px;
}
QPushButton:hover#pesquisar
{
  font-size: 20px;
}
QPushButton#pesquisar_2
{
background-color: rgb(255, 0, 0);
border-radius: 0px;
background: rgb(255, 0, 0);
font-size: 16px;
}
QPushButton:hover#pesquisar_2
{
  font-size: 20px;
}
QPushButton#seca
{
  background-color: rgb(255, 255, 255);
  font-size: 16px;
}
QPushButton:hover#seca
{
  font-size: 20px;
}
QPushButton#zarar
{
   background-color: rgb(255, 170, 0);
   font-size: 16px;
}
QPushButton:hover#zarar
{
  font-size: 20px;
}
QPushButton#zerar_2
{
  background-color: rgb(255, 255, 0);
  border-radius: 0px;
  background-color: rgb(255, 255, 255);
  font-size: 16px;
}
QPushButton:hover#zerar_2
{
  font-size: 20px;
}
QPushButton#calcular_3
{
 background-color: rgb(0, 255, 0);
  border-radius: 0px;
  background: rgb(0, 255, 0);
  font-size: 16px;
}
QPushButton#todas
{
  background-color: rgb(255, 255, 255);
  border-radius: 0px;
  background-color: rgb(128, 128, 128);
  font-size: 16px;
}
QPushButton:hover#todas
{
 font-size: 20px;
}
QPushButton:hover#calcular_3
{
 font-size: 20px;
}


</string>
   </property>
   <widget class="QTableWidget" name="tabela">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1111</width>
      <height>461</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>1600</width>
      <height>900</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 255, 127);
selection-background-color: rgb(200, 147, 13);
font-size: 13px;
text-align: center;</string>
    </property>
    <property name="gridStyle">
     <enum>Qt::NoPen</enum>
    </property>
    <column>
     <property name="text">
      <string>Nome</string>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Numero</string>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Data da ensiminação</string>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Data da Secagem</string>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Data do Parto</string>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Data da nova ensiminação</string>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Nº de Crias</string>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Nº Repetições</string>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>DDL</string>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>440</y>
      <width>1081</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="todas_2">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>600</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>-1</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Todas</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nova_vaca_2">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>600</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>-1</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Começar a secar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="todas">
       <property name="maximumSize">
        <size>
         <width>600</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>-1</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Já deve estar seca</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="calcular_3">
       <property name="maximumSize">
        <size>
         <width>600</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>-1</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>PODE INSIMINAR</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pesquisar_2">
       <property name="maximumSize">
        <size>
         <width>600</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>-1</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>DEVE INSIMINAR</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>500</y>
      <width>1081</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="calcular">
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>-1</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Calcular</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nova_vaca">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>-1</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Nova Vaca</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pesquisar">
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>-1</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Pesquisar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="seca">
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>-1</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Seca</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="parto">
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>-1</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Parto</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deletar">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>-1</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Deletar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="zarar">
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>-1</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Zerar</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="pesquisar_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>0</y>
      <width>1031</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="inputMask">
     <string extracomment="Pesquisar"/>
    </property>
    <property name="placeholderText">
     <string>Pesquisar</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1097</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
